articles/tech-resources/custom-bounce-domain.md

```diff
@@ -16,37 +16,36 @@ To configure a bounce domain, choose the subdomain you would like to use (e.g. m
 
 This DNS record will ensure that out-of-band/asynchronous bounces are routed to SparkPost for processing.
 
-Once you have configured your DNS settings, register and verify the domain with  SparkPost by following the steps below:
+Once you have configured your DNS settings, register and verify the domain with SparkPost by following the steps below:
 
 1. Register the domain with SparkPost by using the sending domains [create endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-create-post)
-1. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
+2. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
```

Comment:
Note: Markdown lets you use `1.` for each numbered list item as a convenience. That makes it easy to re-order and add items later. It will render the list numbers correctly based on the positioning of each item in the list.